src/App.js

```diff
@@ -1,24 +1,47 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
+import { collection, addDoc } from 'firebase/firestore';
+import { db } from './lib/firebase';
 import './App.css';
 
 function App() {
+  const [text, setText] = useState('');
+
+  //handle submit for the form
+  const handleClick = async (e) => {
```

Comment:
how about if we be more specific and call it handleAddItem function or something like that